src/app/guards/auth.guard.ts

```diff
@@ -0,0 +1,28 @@
+import { ActivatedRouteSnapshot, CanActivateFn, RouterStateSnapshot } from "@angular/router";
+import { inject, PLATFORM_ID } from "@angular/core";
+import { isPlatformBrowser } from "@angular/common";
+
+export const canActivateLoggedIn: CanActivateFn = (
```

Comment:
for clearer readability name them something with Guard at the end so it's clear that it is a guard, like

loggedInGuard

isLoggedInGuard



these are too similar so perhaps different name for one of them will be good (this comment just a small thing I noticed)